app/views/lectures/edit/_tutorials.html.erb

```diff
@@ -19,43 +19,48 @@
        aria-labelledby="heading"
        data-parent="#lectureAccordion">
     <div class="card-body">
-      <div class="row mb-3">
-        <div class="col-12 text-center">
-          <%= link_to t('admin.tutorial.new'),
-                      new_tutorial_path(params: { lecture_id: lecture.id }),
-                      class: 'btn btn-sm btn-primary',
-                      id: 'newTutorialButton',
-                      remote: true %>
+      <% if current_user.can_view_users?(lecture) %>
```

Comment:
You disabled the Tutorials section for non-active teachable editors. However, the corresponding POST route `http://localhost:3000/tutorials` is still left open, so these users can continue to create a new tutorial although they cannot see the section in the UI. This might be the case for other tutorial routes as well. Non-active teachable editors should not be able to access those at all.